tests/app/dao/test_users_dao.py

```diff
@@ -36,62 +38,55 @@ def test_create_user(notify_api, notify_db, notify_db_session):
     assert not user.platform_admin
 
 
-def test_get_all_users(notify_api, notify_db, notify_db_session, sample_user):
-    assert User.query.count() == 1
-    assert len(get_user_by_id()) == 1
-    email = "another.notify@digital.cabinet-office.gov.uk"
-    another_user = create_sample_user(notify_db,
-                                      notify_db_session,
-                                      email=email)
+@pytest.mark.usefixtures('notify_db_session')
```

Comment:
this only ever works if you don't have any funcarg method names, otherwise it's not considered, it seems :(